test/legacy/scaleway_ssh_playbook.yml

```diff
@@ -0,0 +1,10 @@
+- name: Test SSH key lifecyle on a Scaleway account
```

Comment:
add other tests:
- with check mode enabled
- check nothing is changed when nothing should have been changed

Reply:
done

Reply:
done